src/stores/UserStore.js

```diff
@@ -69,6 +71,8 @@ export default class UserStore {
   }
 
   getPercentageCompletion(collectionname){
+    // This could be made prettier with a switch. Even better (the only 2 lines)
+    // use a data object for the percentage divider) - Harold
```

Comment:
cool! I love using switch statements.  the data object sounds interesting too. we will see what we can do.